src/main/java/seedu/address/logic/commands/RedoCommand.java

```diff
@@ -25,6 +25,7 @@ public CommandResult execute() throws CommandException {
         }
 
         undoRedoStack.popRedo().redo();
+        model.maintainSorted();
```

Comment:
If the previous list is sorted, why sort it again?

Reply:
Not sure if the redo command will mess up the order